autowarefoundation/autoware
# [feature] Added Autoexposure to Baumer camera node

## Status
**PRODUCTION / DEVELOPMENT**

## Description
This PR solves autowarefoundation/autoware_ai#62 

## Todos


## Steps to Test or Reproduce
1. Pull this branch
1. Make sure you have installed Baumer SDK
1. Make sure you have a camera device available
1. Follow instructions on `README.md` in the package.
1. Launch rviz
1. Add `image_raw`

The camera stream should be visible.
